lib/do_auth/router.ex

```diff
@@ -6,10 +6,10 @@ defmodule DoAuth.Router do
   """
   use Plug.Router
 
-  plug(:match)
-  plug(CORSPlug)
-  plug(Plug.Logger, log: :debug)
-  plug(:dispatch)
+  plug CORSPlug
```

Comment:
I have different (default!) `mix format` settings. If you format like this, you'll have to share the config with us, but in Doma, we're all about reusing as much as we can and configuring as little as we can.